react/features/base/settings/functions.js

```diff
@@ -81,3 +83,17 @@ export function getPropertyValue(
 
     return undefined;
 }
+
+/**
+ * Returns the settings object (or empty object). This is a helper function to
+ * avoid using the redux subtree name every time we need the settings in the
+ * app.
+ *
+ * @public
+ * @param {Object|Function} stateful - The redux state object or
+ * {@code getState} function.
+ * @returns {Object}
+ */
+export function getSettings(stateful: Object | Function): Object {
+    return toState(stateful)['features/base/settings'] || {};
```

Comment:
Why are we returning here an empty object sometimes? I think we should have some reasonable defaults in the redux store.

Reply:
The defaults are actually there in the _initSettings function, and also as we're refactoring this whole flow (slowly) some defaults will come from config.js (e.g. displayName) or other places. This is just an overly defensive was to avoid (perhaps unnecessary) null checks and make me sleep better.

Reply:
I insist. Let's have a proper default state so we don't have to do this. This is the pattern we follow elsewhere, AFAICT.

Reply:
@lyubomir @virtuacoplenny Do you have a list of defaults in mind?

Reply:
You said they already kinda are in `_initSettings` right? So use those here instead?

Reply:
But then that will be executed anyhow, and we'll remove this function too, so I guess this thread is outdated now